In [9]:
import overpy
import pandas as pd
import json
from collections import Counter

In [10]:
api = overpy.Overpass()

In [11]:
#サンプルエリア：浅草周辺
query = """
[out:json][timeout:60];
(
  node["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](35.710,139.790,35.720,139.800);
  way["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](35.710,139.790,35.720,139.800);
  relation["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](35.710,139.790,35.720,139.800);
);
out body;
>;
out skel qt;
"""

In [13]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.get('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 117

ユニークなタグ数: 82

=== 頻出タグ TOP 20 ===
tourism: 17
name: 14
addr:housenumber: 9
name:en: 8
addr:block_number: 8
addr:city: 8
addr:neighbourhood: 8
addr:postcode: 8
addr:province: 7
opening_hours: 6
source: 5
wikidata: 5
name:ja: 5
name:fr: 4
name:es: 4
name:zh: 4
wikipedia: 4
wheelchair: 3
addr:country: 3
image: 3

=== バリアフリー関連タグ ===
wheelchair: 3
wheelchair:description: 1
internet_access: 1
description:en: 1
description:es: 1
description:fr: 1
description:it: 1
description:ja: 1
description:ja-Hira: 1
description:ja-Latn: 1
description:ko: 1
description:ru: 1
description:zh: 1
access: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: 太皷館 ---
  name: 太皷館
  name:en: Drum Museum
  name:fr: Musée du tambour
  note: National-Land Numerical Information (Public Facility) 2006, MLIT Japan
  note:ja: 国土数値情報（公共施設データ）平成19年　国土交通省
  source: KSJ2
  source_ref: http://nlftp.mlit.go.jp/ksj/jpgis/datalist/KsjTmplt-P02-v2_0.html
  tourism: museum
  wikidata: Q104698257

--- 観光地 2: 江戸下町伝統工芸館 ---
 

In [13]:
print(f"観光地（ノード）: {len(result.nodes)}")
print(f"観光地（ウェイ）: {len(result.ways)}")
print(f"観光地（リレーション）: {len(result.relations)}")
print(f"合計: {len(result.nodes) + len(result.ways) + len(result.relations)}")

# 観光地本体のみカウント
tourism_count = sum(1 for n in result.nodes if 'tourism' in n.tags)
tourism_count += sum(1 for w in result.ways if 'tourism' in w.tags)
tourism_count += sum(1 for r in result.relations if 'tourism' in r.tags)
print(f"\ntourismタグを持つ要素: {tourism_count}")

観光地（ノード）: 113
観光地（ウェイ）: 4
観光地（リレーション）: 0
合計: 117

tourismタグを持つ要素: 17


In [15]:
#23区全体
query = """
[out:json][timeout:180];
(
  node["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](35.6,139.6,35.8,139.9);
  way["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](35.6,139.6,35.8,139.9);
  relation["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](35.6,139.6,35.8,139.9);
);
out body;
>;
out skel qt;
"""

In [16]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.ge# tourismタグを持つ要素のみカウント
tourism_elements = [e for e in result.nodes + result.ways + result.relations 
                   if 'tourism' in e.tags]
print(f"tourismタグを持つ要素: {len(tourism_elements)}")

# 余分なノード（wayの構成点など）
extra_nodes = len(result.nodes) - sum(1 for n in result.nodes if 'tourism' in n.tags)
print(f"余分なノード（構成点など）: {extra_nodes}")t('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 6161

ユニークなタグ数: 317

=== 頻出タグ TOP 20 ===
tourism: 1057
name: 819
name:en: 622
wikidata: 278
source: 226
wikipedia: 224
name:ja: 221
opening_hours: 209
website: 202
addr:housenumber: 144
wikimedia_commons: 143
name:es: 141
note: 139
source_ref: 135
wheelchair: 132
building: 132
addr:neighbourhood: 126
addr:block_number: 125
addr:province: 122
addr:city: 119

=== バリアフリー関連タグ ===
wheelchair: 132
description: 16
internet_access: 12
toilets:wheelchair: 12
toilets: 11
internet_access:fee: 5
description:en: 5
description:fr: 5
internet_access:ssid: 4
access: 4
wheelchair:description: 3
description:es: 3
description:ja: 3
description:zh: 3
outdoor_seating: 2
description:ko: 2
internet_access:password: 1
description:it: 1
description:ja-Hira: 1
description:ja-Latn: 1
description:ru: 1
toilets:changing_table: 1
toilets:access: 1
toilets:disposal: 1
toilets:position: 1
indoor: 1
outdoor: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: アーティゾン美術館 ---
  addr:block_number: 7
  addr:housenumber

In [17]:
# tourismタグを持つ要素のみカウント
tourism_elements = [e for e in result.nodes + result.ways + result.relations 
                   if 'tourism' in e.tags]
print(f"tourismタグを持つ要素: {len(tourism_elements)}")

# 余分なノード（wayの構成点など）
extra_nodes = len(result.nodes) - sum(1 for n in result.nodes if 'tourism' in n.tags)
print(f"余分なノード（構成点など）: {extra_nodes}")

tourismタグを持つ要素: 1057
余分なノード（構成点など）: 4992


In [14]:
#群馬
query = """
[out:json][timeout:180];
(
  node["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](36.2,138.7,36.7,139.5);
  way["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](36.2,138.7,36.7,139.5);
  relation["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](36.2,138.7,36.7,139.5);
);
out body;
>;
out skel qt;
"""

In [17]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.get('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 1396

ユニークなタグ数: 95

=== 頻出タグ TOP 20 ===
tourism: 250
name: 198
source: 73
note: 62
note:ja: 61
source_ref: 60
wikidata: 49
building: 43
KSJ2:ADS: 37
KSJ2:PubFacAdmin: 37
name:en: 30
wikipedia: 26
name:ja: 23
website: 15
opening_hours: 13
operator: 13
addr:city: 11
addr:postcode: 11
historic: 11
direction: 11

=== バリアフリー関連タグ ===
wheelchair: 3
toilets: 2
outdoor_seating: 1
toilets:wheelchair: 1
access: 1
description: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: 伊香保保科美術館 ---
  name: 伊香保保科美術館
  tourism: museum

--- 観光地 2: 上里町立図書館・郷土資料館 ---
  name: 上里町立図書館・郷土資料館
  note: National-Land Numerical Information (Public Facility) 2006, MLIT Japan
  note:ja: 国土数値情報（公共施設データ）平成19年　国土交通省
  source: NRPB of MLIT Japan
  source:ja: 国土数値情報（公共施設データ）平成18年　国土交通省
  source_ref: http://nlftp.mlit.go.jp/ksj/jpgis/datalist/KsjTmplt-P02-v2_0.html
  tourism: museum

--- 観光地 3: 本庄市立歴史民俗資料館 ---
  name: 本庄市立歴史民俗資料館
  note: National-Land Numerical Information (Public Facility) 2006, MLIT Japan
  note:ja: 国土数

In [19]:
#福岡
query = """
[out:json][timeout:300];
area["name"="福岡県"]["admin_level"="4"]->.fukuoka;
(
  node(area.fukuoka)["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"];
  way(area.fukuoka)["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"];
);
out body;
>;
out skel qt;
"""

In [21]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.get('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 2176

ユニークなタグ数: 133

=== 頻出タグ TOP 20 ===
tourism: 586
name: 422
source: 157
name:en: 124
wikidata: 67
building: 58
attraction: 54
note: 51
source_ref: 48
note:ja: 47
wikipedia: 42
name:ja: 35
KSJ2:ADS: 30
KSJ2:AdminArea: 30
KSJ2:PubFacAdmin: 30
opening_hours: 30
ele: 21
website: 17
direction: 16
museum: 15

=== バリアフリー関連タグ ===
wheelchair: 8
description: 4
access: 2
description:en: 1
description:ja: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: 観覧車 ---
  attraction: big_wheel
  name: 観覧車
  tourism: attraction

--- 観光地 2: 片江展望台 ---
  covered: no
  description: 油山の中腹に位置する福岡市屈指の夜景の名所として有名です。 自動販売機ゴミ箱と不定期で営業しているホットドッグ屋があり、トイレはありません。 また、春分の日前後にはハチクマというタカ科の大型鳥類の渡りが観察できます。
  fee: no
  name: 片江展望台
  tourism: viewpoint

--- 観光地 3: 新天町商店街 ---
  name: 新天町商店街
  name:en: Shintencho Shopping Arcade
  tourism: attraction


In [24]:
#愛知
query = """
[out:json][timeout:300];
area["name"="愛知県"]["admin_level"="4"]->.aichi;
(
  node(area.aichi)["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"];
  way(area.aichi)["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"];
);
out body;
>;
out skel qt;
"""

In [25]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.get('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 4685

ユニークなタグ数: 122

=== 頻出タグ TOP 20 ===
tourism: 846
name: 590
source: 172
wikidata: 128
name:en: 126
wikipedia: 114
source_ref: 114
note: 112
building: 112
note:ja: 111
attraction: 80
name:ja: 75
wikimedia_commons: 57
operator: 50
opening_hours: 43
KSJ2:ADS: 43
KSJ2:AdminArea: 43
KSJ2:PubFacAdmin: 43
museum: 42
addr:city: 38

=== バリアフリー関連タグ ===
wheelchair: 26
internet_access: 4
access: 3
description: 3
internet_access:fee: 1
toilets:wheelchair: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: 徳川園 ---
  name: 徳川園
  tourism: viewpoint
  wheelchair: yes

--- 観光地 2: 歴史民俗資料館 ---
  contact:phone: +81 536 37 2269
  name: 歴史民俗資料館
  note: 火曜定休 祝日は開館
  opening_hours: 9:00-17:00
  tourism: museum

--- 観光地 3: 鳳来寺山自然科学博物館 ---
  name: 鳳来寺山自然科学博物館
  tourism: museum


In [26]:
#大阪
query = """
[out:json][timeout:300];
area["name"="大阪府"]["admin_level"="4"]->.osaka;
(
  node(area.osaka)["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"];
  way(area.osaka)["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"];
);
out body;
>;
out skel qt;
"""

In [27]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.get('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 4038

ユニークなタグ数: 189

=== 頻出タグ TOP 20 ===
tourism: 747
name: 673
name:en: 236
source: 169
wikidata: 108
wikipedia: 101
building: 99
name:ja: 64
wikimedia_commons: 58
website: 57
attraction: 56
source_ref: 55
note: 53
opening_hours: 49
level: 46
note:ja: 44
historic: 28
wheelchair: 28
shop: 28
fee: 27

=== バリアフリー関連タグ ===
wheelchair: 28
description: 12
toilets:wheelchair: 4
internet_access: 4
access: 3
toilets: 2
internet_access:fee: 1
indoor: 1
description:en: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: 大阪城天守閣 ---
  access: permissive
  alt_name:es: Castillo Osaka
  alt_name:ja_rm: Ōsaka-jō
  castle_type: shiro
  fee: ¥700
  historic: castle
  historic:civilization: medieval
  name: 大阪城天守閣
  name:de: Burg Ōsaka
  name:en: Osaka Castle
  name:es: Castillo de Osaka
  name:fr: Château d'Osaka
  name:it: Castello di Osaka
  name:ja: 大阪城天守閣
  name:ja_kana: オオサカジョウテンシュカク
  name:ja_rm: Oosaka-jou Tenshukaku
  name:ko: 오사카 성
  name:nl: Kasteel Osaka
  name:pt: Castelo de Osaka
  name

In [28]:
#神戸
query = """
[out:json][timeout:180];
(
  node["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](34.6,135.1,34.8,135.3);
  way["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](34.6,135.1,34.8,135.3);
  relation["tourism"~"museum|attraction|gallery|theme_park|viewpoint|zoo|aquarium"](34.6,135.1,34.8,135.3);
);
out body;
>;
out skel qt;
"""

In [29]:
print("クエリ実行中...")
result = api.query(query)

# すべてのタグを収集
all_tags = []
for element in result.nodes + result.ways + result.relations:
    all_tags.extend(element.tags.keys())

# タグの出現頻度
tag_counts = Counter(all_tags)

print(f"\n取得した観光地数: {len(result.nodes) + len(result.ways) + len(result.relations)}")
print(f"\nユニークなタグ数: {len(tag_counts)}")
print("\n=== 頻出タグ TOP 20 ===")
for tag, count in tag_counts.most_common(20):
    print(f"{tag}: {count}")

# バリアフリー関連タグを抽出
bf_tags = {k: v for k, v in tag_counts.items() 
           if any(keyword in k.lower() for keyword in 
                  ['wheelchair', 'elevator', 'ramp', 'toilet', 'door', 
                   'access', 'tactile', 'description'])}

print("\n=== バリアフリー関連タグ ===")
for tag, count in sorted(bf_tags.items(), key=lambda x: x[1], reverse=True):
    print(f"{tag}: {count}")

# サンプルデータの詳細確認
print("\n=== サンプル観光地の全タグ ===")
for i, node in enumerate(result.nodes[:3]):  # 最初の3件
    print(f"\n--- 観光地 {i+1}: {node.tags.get('name', 'Unknown')} ---")
    for key, value in node.tags.items():
        print(f"  {key}: {value}")

クエリ実行中...

取得した観光地数: 1256

ユニークなタグ数: 102

=== 頻出タグ TOP 20 ===
tourism: 193
name: 140
name:en: 72
wikidata: 65
building: 65
wikipedia: 60
wikimedia_commons: 43
source: 31
note: 22
website: 21
source_ref: 20
note:ja: 19
historic: 19
alt_name: 18
addr:city: 11
addr:block_number: 10
addr:suburb: 10
opening_hours: 10
name:ja: 10
addr:postcode: 9

=== バリアフリー関連タグ ===
wheelchair: 6
description: 2
toilets: 1

=== サンプル観光地の全タグ ===

--- 観光地 1: 東六甲展望台 ---
  name: 東六甲展望台
  tourism: viewpoint

--- 観光地 2: Unknown ---
  name:en: Tenrandai Observatory
  tourism: viewpoint

--- 観光地 3: 切手文化博物館 ---
  name: 切手文化博物館
  note: National-Land Numerical Information (Public Facility) 2006, MLIT Japan
  note:ja: 国土数値情報（公共施設データ）平成19年　国土交通省
  source: KSJ2
  source_ref: http://nlftp.mlit.go.jp/ksj/jpgis/datalist/KsjTmplt-P02-v2_0.html
  tourism: museum
  wikidata: Q11516762
  wikipedia: ja:有馬切手文化博物館


| 都市     |   観光施設数 |   wheelchair情報 | BF対応率   |   toilets:wheelchair |
|:---------|-------------:|-----------------:|:-----------|---------------------|
| 東京23区 |         1057 |              132 | 12.5%      |                   12 |
| 大阪府   |          747 |               28 | 3.7%       |                    4 |
| 愛知県 |          846 |               26 | 3.1%       |                    1 |
| 福岡県   |          586 |                8 | 1.4%       |                    0 |
| 神戸市   |          193 |                6 | 3.1%       |                    0 |
| 群馬県   |          250 |                3 | 1.2%       |                    1 |